stack.yaml

```diff
@@ -26,7 +26,7 @@ extra-deps:
       - cardano-crypto-class
 
   - git: https://github.com/input-output-hk/cardano-ledger
-    commit: 344aa563be42f6a2682f5b7ff1e4978e4d75359a
+    commit: f6ee4db0a88b2cb74c9e8a2664a8a7fd6fd50c26
```

Comment:
TODO fix this when https://github.com/input-output-hk/cardano-ledger/pull/647 is merged.